TESTS/mbed_drivers/ticker/main.cpp

```diff
@@ -71,7 +71,9 @@ void ticker_callback_1(void)
 void ticker_callback_2(void)
 {
     ++callback_trigger_count;
-    switch_led2_state();
+    if (LED2 != NC) {
+        switch_led2_state();
```

Comment:
Can you send this as separate PR - fixing this LED2 = NC condition.

Reply:
TESTS/mbed_drivers/ticker/main.cpp change is now submitted as https://github.com/ARMmbed/mbed-os/pull/10976